parser/statement.go

```diff
@@ -36,12 +37,13 @@ func (self *_parser) parseStatementList() (list []ast.Statement) {
 }
 
 func (self *_parser) parseStatement() ast.Statement {
-
 	if self.token == token.EOF {
 		self.errorUnexpectedToken(self.token)
 		return &ast.BadStatement{From: self.idx, To: self.idx + 1}
 	}
 
+	allowImportExport := self.scope.allowImportExport
+	self.scope.allowImportExport = false
```

Comment:
This is likely the first hack I needed, and it is the one I haven't figured out any easier way - I am really hopeful I just missed something. 

Maybe I should just have "are we in the top scope" boolean that works in the same way and combine it with `allowImportExport` instead ?